bokehjs/examples/stocks/stocks.ts

```diff
@@ -0,0 +1,73 @@
+console.log(`Bokeh ${Bokeh.version}`);
+Bokeh.set_log_level("debug");
+
+
+function make_plot(title:string, source:Bokeh.ColumnDataSource) {
```

Comment:
I was hope this annoying prefixing can be removed at some point. Thought I didn't find a way for TS to do this.